comps/dataprep/redis/multimodal_langchain/prepare_videodoc_redis.py

```diff
@@ -0,0 +1,533 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import shutil
+import subprocess
+import time
+import uuid
+from pathlib import Path
+from typing import Any, Dict, Iterable, List, Optional, Type, Union
+
+from config import EMBED_MODEL, INDEX_NAME, INDEX_SCHEMA, LVM_ENDPOINT, REDIS_URL, WHISPER_MODEL
+from fastapi import File, HTTPException, UploadFile
+from langchain_community.utilities.redis import _array_to_buffer
+from langchain_community.vectorstores import Redis
+from langchain_community.vectorstores.redis.base import _generate_field_schema, _prepare_metadata
+from langchain_community.vectorstores.redis.schema import read_schema
+from langchain_core.embeddings import Embeddings
+from langchain_core.utils import get_from_dict_or_env
+from langsmith import traceable
```

Comment:
please remove langsmith since OPEA has removed it.

Reply:
Thank for your feedback. We have removed langsmith here.